front/app/containers/MainHeader/Components/TenantLogo/index.tsx

```diff
@@ -30,15 +28,20 @@ const Logo = styled(Image)`
   cursor: pointer;
 `;
 
-const TenantLogo = ({ intl: { formatMessage } }: WrappedComponentProps) => {
+const TenantLogo = ({ ...props }: BoxProps) => {
```

Comment:
I wanted to be able to override the "flex" prop - so I figured allowing all the BoxProps was the best solution? :thinking: Maybe I'm wrong though :thinking:

Reply:
The default value it has makes it take up too much vertical space on the full menu compared to Rob's design :)